api/generator/generator.go

```diff
@@ -275,6 +284,15 @@ func (cg *CertGeneratorHandler) Handle(w http.ResponseWriter, r *http.Request) e
 		},
 	}
 
+	if req.Bundle && cg.bundler != nil {
```

Comment:
@kisom Updated to send error message. I wasn't sure of the exact best type, so I just went with (`PolicyError`, `InvalidRequest`)